LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
Well we had this both on our website (from where the screenshot it taken) and most documents.  

![image](https://user-images.githubusercontent.com/673121/148295125-6d7d8b95-038d-42bc-8e7f-88353c3529e6.png)

Reply:
@eddelbuettel ah good catch! I've changed the license to GPL.

Reply:
:heart:

Reply:
Yeah absolutely. It wasn't intentional, I had just forgotten.